hash/msx2_flop.xml

```diff
@@ -8164,6 +8193,99 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="konaish1">
+		<description>Konai Shasei Vol. 1 (Japan)</description>
+		<year>1991</year>
+		<publisher>Fairytale</publisher>
+		<info name="alt_title" value="校内写生１巻"/>
```

Comment:
“Kounai”

Reply:
The Japanese boxes for these software titles have "Konai Shasei" which is why I chose to use Konai.